app/assets/javascripts/views/templates/servers/suite_select.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="test-run-result" id="test-{{suite.id}}">
+<div class="test-run-result" id="test-{{suite.id}}" title="{{suite.title}}">
```

Comment:
let's not add a title, let's bind the test object to the html element using data so that we can filter by more than just title